server/hotrod/src/main/java/org/infinispan/server/hotrod/BaseRequestProcessor.java

```diff
@@ -82,7 +83,8 @@ void writeException(HotRodHeader header, Throwable cause) {
             status = OperationStatus.ServerError;
          }
       } else if (header != null) {
-         log.exceptionReported(cause);
+         if (cause instanceof MissingMembersException) log.warn(cause.getMessage());
```

Comment:
I feel like logging that a member is missing as WARN is a bit too high level. This is a valid state if another server is still starting back up.  I would suggest lowering the severity of logging level for this.

Reply:
Also side note but we can't use an unknown cause message for WARN as it needs to be i18n supported.

Reply:
I added an additional check here for tracing, I think using ERROR would pollute the log too much.